bueno/fmt.rs

```diff
@@ -0,0 +1,80 @@
+use deno_core::error::AnyError;
+use dprint_plugin_json;
+use dprint_plugin_markdown;
+use dprint_plugin_typescript;
+use glob::glob;
+use std::ffi::OsStr;
+use std::path::{Path, PathBuf};
+
+fn fake_path(ext: &str) -> PathBuf {
+    let file_name = format!("file.{}", ext);
+    PathBuf::from(file_name)
+}
+
+fn format_typescript_file(path: &Path, contents: &str) -> Result<Option<String>, AnyError> {
+    dprint_plugin_typescript::format_text(
+        path,
+        contents,
+        &dprint_plugin_typescript::configuration::ConfigurationBuilder::new()
```

Comment:
Shouldn't we just pass around the dprint configuration instead of creating new one every time?

Reply:
Added TODO, I think this is a future thing though.